api-gateway/src/main/resources/static/scripts/owner-details/owner-details.template.html

```diff
@@ -31,10 +31,10 @@
     <tr>
         <td>
             <a class="newBtn btn-default btn" style="background-color: royalblue; color: white" ui-sref="ownerEdit({ownerId: $ctrl.owner.id, method: 'edit'})">Edit Owner</a>
+            <a class="newBtn btn-default btn btn-danger" style="background-color: #dc3545;" ng-click = "$ctrl.deleteOwner()">Delete Owner</a>
```

Comment:
You changed the location of the button? why?

Reply:
That was an accident, when originally not meant to be moved but I did because I wanted to add an it to another area, remembered it meant to be staying local to "Owner Details", forgot the original position of it.

Reply:
However it does not interfere with anything else